web/src/templates/newsTemplate.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import PropTypes from 'prop-types';
+
+import Layout from '../components/layout';
+import Hero from '../components/hero';
+import NewsContent from '../components/newsContent';
+import RecentNewsList from '../components/recentNewsList';
+
+export default function NewsTemplate({ data }) {
+    const {
+        newsItem,
+        news: { edges },
+    } = data;
+
+    const { heroImage, newsTitle } = newsItem;
+
+    const posts = edges
+        .map((post) => post.node)
+        .filter((post) => post.id !== newsItem.id);
```

Comment:
`filter` creates a new array so I don’t think you need `map`. Stick with the `filteredPosts` naming convention so it’s clear what this array is. 

Also, destructure `id` from `newsItem`

Reply:
@asasmith that didn't work for me when I tried over the weekend. it was not pulling anything out from the array

Reply:
`.filter((post) => post.node.id !== newsItem.id` should work. 

You'll also have to adjust `recentNewsList.js` to destructure `node` in the `map` method (similar to `newsList.js` [here](https://github.com/bckrm/airtec/blob/f468dfab9cbcc6ee393f2aa8f384a711d7e33356/web/src/components/newsList.js#L56-L59))